pylib/anki/collection.py

```diff
@@ -972,6 +972,16 @@ def set_aux_template_config(
         )
         return self.set_config(key, value, undoable=undoable)
 
+    def _get_enable_load_balancer(self) -> bool:
+        return self.get_config_bool(Config.Bool.LOAD_BALANCER_ENABLED)
+
+    def _set_enable_load_balancer(self, value: bool) -> None:
+        self.set_config_bool(Config.Bool.LOAD_BALANCER_ENABLED, value)
```

Comment:
Since this change, I get these two type errors from `mypy`:

```
pylib/anki/collection.py:976: error: "type[Bool]" has no attribute "LOAD_BALANCER_ENABLED"  [attr-defined]
pylib/anki/collection.py:979: error: "type[Bool]" has no attribute "LOAD_BALANCER_ENABLED"  [attr-defined]
```

Do you get them too @jakeprobst? Or is this a misconfiguration on my personal system?

Reply:
regen your protos (probably)

Reply:
How do I do that?